docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,10 +1,220 @@
 [[get-started-docker]]
```

Comment:
I wasn't able to run the non-secured example with the default docker for mac settings (2 CPUs and 2 GiB Memory) without at least one of the nodes failing. I bumped up the allocation to 4 CPU and 4 GiB Memory (Docker for Mac -> Settings -> Advanced) and tested and it appears to work fine. I'm thinking we might want to add a prerequisite section that outlines the resource requirements for running these configurations (while my test worked, it would be good to have someone from the ES team weigh in on optimal recommendations).

Reply:
I agree we should clarify that the user needs to check that there's sufficient memory in Docker for Mac (I suppose also for Windows) and we recommend at least 4GiB -- given that these examples have 512MiB heap (plus half of that for Max Direct Memory) x 3 + enough memory for Kibana. Also we should include the Docker link that it can be changed via https://docs.docker.com/docker-for-mac/#resources

Reply:
In my own testing, once I bumped to 4 GiB things worked fine. Agree that needs to be called out.